plugins/privacy/user/user.php

```diff
@@ -149,6 +149,30 @@ public function onPrivacyRemoveData(PrivacyTableRequest $request, JUser $user =
 				->delete($this->db->quoteName('#__session'))
 				->where($this->db->quoteName('session_id') . ' IN (' . implode(', ', $quotedIds) . ')')
 		)->execute();
+
+				// Remove sensible user custom field data
```

Comment:
Please remove the spaces :-)
```suggestion
		// Remove sensible user custom field data
```

Reply:
Change `sensible` to `sensitive`.